src/main/java/pl/allegro/tech/embeddedelasticsearch/InstanceDescription.java

```diff
@@ -1,19 +1,30 @@
 package pl.allegro.tech.embeddedelasticsearch;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
 class InstanceDescription {
-    private final int port;
-    private final String clusterName;
+    private Map<String, Object> settings;
```

Comment:
The idea was allow calls as:
``
.withSetting("some.boolean.setting", true);
``

Anyway I will change it to String.